src/core/colorspace.js

```diff
@@ -771,8 +772,8 @@ var CalGrayCS = (function CalGrayCSClosure() {
 
     // Validate variables as per spec.
     if (this.XW < 0 || this.ZW < 0 || this.YW !== 1) {
-      error('Invalid WhitePoint components for ' + this.name +
-            ', no fallback available');
+      throw new FormatError('Invalid WhitePoint components for ' + this.name +
```

Comment:
Unable to use that since string must be broken in two line, and lint errors with "Strings must use singlequote"